Rakefile

```diff
@@ -266,6 +266,19 @@ agents = [
             'yum install -y java-1.8.0-openjdk-headless git mercurial subversion openssh-clients bash unzip curl',
             'yum clean all'
         ]
+    },	
+    {
+        distro: 'fedora',
+        version: '29',
+        release_name:'29',
+        eol_date: '2019-11-30',
+        add_files: tini_and_gosu_add_file_meta,
+        create_user_and_group: create_user_and_group_cmd,
+        before_install: [
+            'yum update -y',
+            'yum install -y java-1.8.0-openjdk-headless git mercurial subversion openssh-clients bash unzip curl',
```

Comment:
LGTM. I can see `java-11-openjdk-headless` being available on fedora, can we use that instead?